### DISCUSSION TITLE
Can I get only the image for the result of the input?

### DISCUSSION THREAD

**Discussion by kimcoldwater** [FreeRDP/FreeRDP#8763] [Q&A]
Hello, I'm making a project using the sample client of freerdp.

Can't I get other values than the image as the result of the operation such as keyboard or mouse input?

For example, can I know information about a specific application being manipulated?

**Comment by akallabeth** [comment]
`-v`? sorry, the question is too vague, what do you want?

**Reply by kimcoldwater** [reply]
I wonder if you can find out whether a specific process (application) of the connected session is activated or not.

Like using Win32.
